### PR TITLE
[17.0][ADD] quality_control_oca_timesheet: New module quality_control_oca_timesheet

### PR DESCRIPTION
**Pull request by ppyczko**
This module integrates timesheet tracking with quality control inspections. It adds a "Timesheets" tab to the quality control inspection form, allowing users to log and track the time spent on each inspection through timesheet entries. 

cc https://github.com/APSL 161856

@miquelalzanillas @lbarry-apsl @mpascuall @peluko00 @javierobcn @BernatObrador please review

### CODE REVIEW

File: quality_control_oca_timesheet/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+When creating or editing a quality control inspection, you will find a new Timesheets tab within the inspection form. In this tab, you can log and track the time spent on the inspection by adding timesheet entries. Each entry allows you to specify the project, task, date, and the time spent on the inspection in hours and minutes.
+
+When creating or editing a quality control inspection, you will find a new Timesheets tab within the inspection form. In this tab, you can log and track the time spent on the inspection by adding timesheet entries. Each entry allows you to specify the project, task, date, and the time spent on the inspection in hours and minutes.
```
Comment (BernatObrador) [inline quality_control_oca_timesheet/readme/USAGE.md:1]: There is a line repeated!

**Review by BernatObrador — APPROVED** [review]
But BTW LGTM code review and tested in runboat

**Review by peluko00 — APPROVED** [review]
LGTM, tested in runboat and works as expected

File: quality_control_oca_timesheet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Patryk Pyczko (APSL-Nagarro)<ppyczko@apsl.net>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Quality Control - Timesheet (OCA)",
+    "version": "17.0.1.0.0",
+    "category": "Quality Control",
+    "website": "https://github.com/OCA/manufacture",
+    "author": "Patryk Pyczko, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline quality_control_oca_timesheet/__manifest__.py]: Here goes the company, not the contributor

File: quality_control_oca_timesheet/i18n/ca_ES.po
Comment (pedrobaeza) [inline quality_control_oca_timesheet/i18n/ca_ES.po:1]: File should be ca.po

**Review by mpascuall — APPROVED** [review]
Once you've fixed the previous reviews, it LGTM tested in runboat

File: quality_control_oca_timesheet/readme/USAGE.md (hunk 1 above)
Comment (ppyczko) [reply quality_control_oca_timesheet/readme/USAGE.md:1]: Ups! Thanks for pointing it out, solved!

File: quality_control_oca_timesheet/__manifest__.py (hunk 2 above)
Comment (ppyczko) [reply quality_control_oca_timesheet/__manifest__.py]: Thank you for letting me know, changed!

File: quality_control_oca_timesheet/i18n/ca_ES.po
Comment (ppyczko) [reply quality_control_oca_timesheet/i18n/ca_ES.po:1]: Done!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**